docs/source/user-guide/dataset_object.rst

```diff
@@ -3,76 +3,74 @@
-Some of the checks allows to use a dataframe directly, but some others requires the metadata in order to run, so they are limited to working only with Datasets.
-
-Class Parameters
-===================
-All of the parameters are optional.
-
-.. list-table::
-    :widths: 20 20 50 10
-    :header-rows: 1
-
-    * - Name
-      - Type
-      - Description
-      - Default
-    * - label
-      - pandas.Series
-      - Data of labels as separate series from the data
-      - None
-    * - features
-      - List[Hashable]
-      - Names of the features in the data
-      - None
-    * - cat_features
-      - List[Hashable]
-      - Names of the categorical features in the data. Must be subset of `features`
-      - None
-    * - label_name
-      - Hashable
-      - Name of label column in the data
-      - None
-    * - use_index
-      - bool
-      - If data is dataframe, whether to use the dataframe index as index column for index related checks
-      - False
-    * - index_name
-      - Hashable
-      - Name of index column in the data
-      - None
-    * - date_name
-      - Hashable
-      - Name of date column in the data
-      - None
-    * - date_unit_type
-      - str
-      - Unit to convert date column if it's numeric. using `pandas.Timestamp <https://pandas.pydata.org/docs/reference/api/pandas.Timestamp.html>`__ to convert
-      - None
-    * - max_categorical_ratio
-      - float
-      - Used to infer which columns are categorical (if `cat_features` isn't explicitly passed).
-        Set maximum ratio of unique values in a column in order for it to be categorical.
-        The higher the value, the chance of column inferred as categorical is higher
-      - 0.01
-    * - max_categories
-      - int
-      - Used to infer which columns are categorical (if `cat_features` isn't explicitly passed).
-        Set maximum number of unique values in a column in order for it to be categorical.
-        The higher the value, the chance of column inferred as categorical is higher
-      - 30
-    * - max_float_categories
-      - int
-      - Same as `max_categories` but for columns of type float
-      - 5
-    * - convert_date
-      - bool
-      - Whether to convert date column if it's numeric to date
-      - True
+
+
+A Quick Introduction
+====================
+
+The Dataset Object is a simple and robust Object to store tabular data and metadata relevant to Machine Learning.
+It allows deepchecks to understand the context of the data with minimal hassle for the user.
+
+Common information used in Machine Learning that are aggregated in the Dataset:
+ - **cat_features** - Categorical features normally require some preprocessing before being passed to the model. If not specified, the categorical features are inferred automatically from the data itself. (`Inferring Features And Categorical Features`_ )
+ - **label** - The classes of the classification problem or values of a regression problem.
```

Comment:
put this one first.
phrasing should be clear and short (that this represents the target that we're trying to predict in the data. 
(more high level, you wrote it very technical)